heapster-saw/src/Verifier/SAW/Heapster/SAWTranslation.hs

```diff
@@ -1975,61 +1994,72 @@ getPermStackDistPerms =
        `mbApply`
        permTransCtxPerms prxs stack
 
+-- | Run a computation if the current 'ChecksFlag' is set
+ifChecksFlagM :: ImpTransM ext blocks tops ret ps ctx () ->
+                 ImpTransM ext blocks tops ret ps ctx ()
+ifChecksFlagM m =
+  (itiChecksFlag <$> ask) >>= \checks ->
+  if unChecksFlag checks then m else return ()
```

Comment:
Nitpick: Perhaps rename `unChecksFlag` to `checksFlagSet`? I think it would make lines like this easier to read.

Reply:
Yes, that makes sense